src/wlapplication.h

```diff
@@ -272,6 +272,10 @@ struct WLApplication {
 	/// Prevent toggling fullscreen on and off from flickering
 	uint32_t last_resolution_change_;
 
+	/// Content for popup error messagebox
+	std::string messagetitle;
+	std::string message;
```

Comment:
`message` and `messagetitle` could be parameters to the `WLApplication::mainmenu()` call instead of member variables. If you think they should be member variables, add underscores to their names.

Reply:
`WLApplication::mainmenu()` will be removed soon, so local variables and local handling of error messages would probably be better. I'm not looking forward to the merge conflicts if this is merged before #4518… ;)

Reply:
Local variables sounds good. I will wait for #4518 and then handle the merge.